### ISSUE TITLE
long single-line from-style imports with hanging grid still use "\"

### ISSUE THREAD

**Issue by ezarowny** [timothycrosley/isort#267]
With multi_line_output=4 I get this:

```
-from notifications.management.commands._base import SendSystemNotificationCommand
+from notifications.management.commands._base import SendSystemNotificationCommandimport \
```

When I think it should be

```
-from notifications.management.commands._base import SendSystemNotificationCommand
+from notifications.management.commands._base import(
+    SendSystemNotificationCommandimport)
```

Is this intended behavior?

**Comment by patrickyevsukov** [comment]
Good news, @ezarowny ! This was fixed in pull request #264. My fork of isort already has the fix merged in. It's available here: https://github.com/truveris/isort

**Comment by ezarowny** [comment]
Thanks man! Much appreciated.

**Comment by patrickyevsukov** [comment]
Glad you found it useful! Have a look at my changes here: truveris@9cc3bdf Using that as a guide, it should be fairly straight forward to fork and alter isort for further one module import formatting customization.

**Comment by timothycrosley** [comment]
This has been fixed officially in isort 4.0 that was just released to PYPI

Thanks!

~Timothy
